src/featureFlags.js

```diff
@@ -99,4 +110,28 @@ export class FeatureFlags extends events.EventEmitter {
   static async clearIgnoreWalletServiceFlag() {
     await localStorage.removeItem(IGNORE_WALLET_SERVICE_FLAG);
   }
+
+  /**
+   * Uses the Hathor Unleash Server and Proxy to determine if the
+   * wallet should have the Atomic Swap feature
+   *
+   * @return {boolean} The result from the unleash feature flag
+   */
+  async shouldUseAtomicSwap() {
```

Comment:
I think the name is not good here. You are not checking if you are using the atomic swap, actually you are checking if atomic swap is enabled, so maybe `isAtomicSwapEnabled` would be better, what do you think?

If we decide to change here, you should also update the names in the actions and reducers.

Reply:
This is a good name. Refactored on [7128f6e](https://github.com/HathorNetwork/hathor-wallet/pull/361/commits/7128f6efb3b081a3c3b10d647f89d3d1c282e4d4)